packages/realm/bindgen/src/templates/jsi.ts

```diff
@@ -912,6 +912,12 @@ class JsiCppDecls extends CppDecls {
             )
             .join("\n")}
 
+
+          // Silence these warnings as they might be confusing to developers as they build the binding
+
+          #pragma GCC diagnostic push
+          #pragma GCC diagnostic ignored "-Wswitch"
```

Comment:
Adding this to silence warnings that might be confusing to our users.

Reply:
But will we accidentally overlook something?